integration/tests/cook/util.py

```diff
@@ -105,20 +105,61 @@ def minimal_job(**kwargs):
     return job
 
 
-def submit_job(cook_url, **kwargs):
-    job_spec = minimal_job(**kwargs)
-    request_body = {'jobs': [job_spec]}
+def minimal_jobs(job_count, **kwargs):
+    """Build a list of of multiple homogeneous job specifications"""
+    return [ minimal_job(**kwargs) for _ in range(job_count) ]
+
+
+def submit_jobs(cook_url, job_specs, clones=1, **kwargs):
+    """
+    Create and submit multiple jobs, either cloned from a single job spec,
+    or specified individually in multiple job specs.
+    Arguments can be manually passed to the scheduler post via kwargs.
+    """
+    if isinstance(job_specs, dict):
+        job_specs = [job_specs] * clones
+    jobs = [ minimal_job(**spec) for spec in job_specs ]
+    request_body = {'jobs': jobs}
+    request_body.update(kwargs)
+    logger.info(request_body)
     resp = session.post('%s/rawscheduler' % cook_url, json=request_body)
-    return job_spec['uuid'], resp
+    return [ j['uuid'] for j in jobs ], resp
+
+
+def cancel_jobs(cook_url, jobs, assert_response=False):
```

Comment:
Can you remove the `kill_job` function in this PR and replace the call with one to `cancel_jobs`?

Reply:
✅